routes/api/ttshare.js

```diff
@@ -0,0 +1,67 @@
+import Student from "../../models/Student";
+import express from 'express';
+import TimeTable from "../../models/TimeTable";
+import mongoose from 'mongoose';
+//import { check, validationResult } from "express-validator";
+const router = express.Router();
+
+
+let ObjectId = mongoose.Types.ObjectId;
+router.get("/shareTT", async (req, res) => {
+    let StudentIDs = req.query.id;
+    //console.log(StudentIDs);
+    //console.log(isNaN(StudentIDs));
+    if (StudentIDs === undefined || StudentIDs.length === 0) {
+        res.send(null);
+    }
+    //let year = req.query.year;
+    else {
+        if (StudentIDs === null) {
+
+            console.log("m8, null");
+            res.send(null);
+        }
```

Comment:
You can add this to the validation check above.
P.S. I still think it would be better to use `express-validator`

Reply:
done